### DISCUSSION TITLE
Consulta de NFS-e

### DISCUSSION THREAD

**Discussion by ferrarih** [TadaSoftware/PyNFe#229] [General]
Pessoal,

Primeira vez que estou tentando utilizar o PyNFe, porém estou com uma dificuldade.
Estou tentando consultar todas as notas emitidas pelo meu CNPJ, as notas são NFSe e estou utilizando o código exemplo abaixo:

https://github.com/TadaSoftware/PyNFe/wiki/Consulta-NFSe-Ginfes

Porém estou recebendo o erro abaixo, sabem o que é?

`    nfse = serializador.consultar_rps(emitente, numero, serie, tipo)
  File "/home/thv_ferrari_systems/my_env/lib/python3.9/site-packages/pynfe/processamento/serializacao.py", line 1297, in consultar_rps
    from pynfe.processamento.autorizador_nfse import SerializacaoGinfes
  File "/home/thv_ferrari_systems/my_env/lib/python3.9/site-packages/pynfe/processamento/autorizador_nfse.py", line 1, in <module>
    from pyxb import BIND
ModuleNotFoundError: No module named 'pyxb'`

**Comment by juniortada** [comment]
@ferrarih o sistema não encontrou instalada a lib pyxb. Provavelmente você não instalou as dependências para NFS-e.
https://github.com/TadaSoftware/PyNFe/blob/main/requirements-nfse.txt

No entanto NFS-e está muito desatualizado, depende muito da versão de outras libs instaladas e o autorizador/cidade 
para a qual você irá usar.
